WordOfTheDay.Domain/WordsServices.cs

```diff
@@ -7,40 +7,44 @@
 
 namespace WordOfTheDay.Domain
 {
-    public static class WordsServices
+    public class WordsServices : IWordsServices
     {
+        private readonly IWordsRepository _iWordsRepository;
+        public WordsServices (IWordsRepository iWordsRepository)
+        {
+            _iWordsRepository = iWordsRepository;
+        }
         
-        public static  Task<WordCount> WordOfTheDay(WordContext context)
+        public Task<WordCount> WordOfTheDay()
         {
-            var wordOfTheDay = WordsRepository.WordOfTheDay(context);
+            var wordOfTheDay = _iWordsRepository.WordOfTheDay();
 
             return wordOfTheDay;
         }
         
-        public static async Task<List<WordCount>> CloseWords(WordContext context, string word)
+        public async Task<List<WordCount>> CloseWords(string word)
         {
-            var closeWords = WordsRepository.CloseWords(context, word);
+            var closeWords = _iWordsRepository.CloseWords(word);
             var closeWordCounts = new List<WordCount>();
 
             foreach (var w in closeWords)
             {
-                closeWordCounts.Add(new WordCount(w.Text, await WordsRepository.CountWord(context, w.Text)));
+                closeWordCounts.Add(new WordCount(w.Text, await _iWordsRepository.CountWord(w.Text)));
```

Comment:
I didn't pay attention on this in you previous PR. But can you fix it in this PR? Can you do all these logic as one DB request? You have already done similar for WordOfTheDay request